tests/flow/test_concurrent_query.py

```
@@ -0,0 +1,422 @@
+            # 1. Rename is done before query is sent. The name in the graph context is new_graph,
+            #    so when upon commit, when trying to open new_graph key,
+            #    it will encounter an empty key since new_graph is not a valid key.
+            #    Note: As from https://github.com/RedisGraph/RedisGraph/pull/820
+            #          this may not be valid since the rename event handler might actually rename the graph key, before the query execution.    
+            # 2. Rename is done during query executing, so when commiting and comparing stored graph context name (GRAPH_ID) to the retrived value graph context name (new_graph),
+            #    the identifiers are not the same, since new_graph value is now stored at GRAPH_ID value.
+
+            result = results[0]
+            if type(result) is ResponseError:
+                possible_exceptions = ["Encountered different graph value when opened key " + GRAPH_ID,
+                                       "Encountered an empty key when opened key " + new_graph]
```

Ruff (0.12.2)

`new_graph` may be undefined, or defined from star imports

(F405)